package.json

```diff
@@ -31,6 +30,6 @@
   },
   "dependencies": {
     "@types/react": "^15.0.21",
```

Comment:
Rule of thumb: if one of the things you expose ends up having a type as part of its definition, you need the types package as a dependency. In this case, your debounce function has the following signature: `debounce(wait?: number, options?: DebounceSettings)`, so it needs to have the package that provides debounce settings for its consumers.